backend-project/small_eod/letters/views.py

```diff
@@ -12,3 +25,31 @@ class LetterViewSet(viewsets.ModelViewSet):
 class DescriptionViewSet(viewsets.ModelViewSet):
     queryset = Description.objects.all()
     serializer_class = DescriptionSerializer
+
+
+class FileViewSet(NestedViewSetMixin,
+                  mixins.CreateModelMixin,
+                  mixins.RetrieveModelMixin,
+                  viewsets.GenericViewSet):
```

Comment:
Ja bym tu użył po prostu ```ModelViewSet```, chociaż dokumentacja tego nie przewidywała, ale napisałem w taki sposób, aby dać zadość Twoim testom.  W oczywisty sposób brakuje w dokumentacji usuwania plików. Lista plików...nie zaszkodzi.

Reply:
Zmieniłem dziedziczenie na ModelViewSet, potrzebujemy testów dodatkowe widoki ?

Reply:
Staramy się to testować w sposób zbiorczy. Zobacz na https://github.com/watchdogpolska/small_eod/blob/dev/backend-project/small_eod/cases/tests/test_views.py#L13-L19 .

Reply:
Przeanalizuje jak zostało to wykonane dla cases i zrobię podobnie dla files.